packages/amplify-e2e-core/package.json

```diff
@@ -22,12 +22,16 @@
     "clean": "rimraf ./lib"
   },
   "dependencies": {
+    "@aws-amplify/graphql-transformer-core": "^1.2.1",
+    "@aws-sdk/client-ec2": "^3.226.0",
+    "@aws-sdk/client-rds": "^3.226.0",
     "amplify-headless-interface": "^1.17.3",
     "chalk": "^4.1.1",
     "execa": "^5.1.1",
     "fs-extra": "^8.1.0",
     "graphql-transformer-core": "8.0.3",
     "jest-environment-node": "^26.6.2",
+    "knex": "~2.3.0",
```

Comment:
I think we recently bumped this to `2.4.0` for a security alert. [Refer](https://github.com/aws-amplify/amplify-category-api/pull/1440/files)

Reply:
Bad rebase, will fix.